docs/apache-airflow-providers-amazon/operators/redshift/redshift_sql.rst

```diff
@@ -35,7 +35,7 @@ Operators
 Execute a SQL query
 ===================
 
-The generic ``SQLExecuteQueryOperator`` can be used to execute SQL queries against an Amazon Redshift cluster using a `Redshift connection <../../connections/redshift.rst>`_.
+The generic ``SQLExecuteQueryOperator`` can be used to execute SQL queries against an Amazon Redshift cluster using a `Redshift connection <../../connections/redshift.html>`_.
```

Comment:
Better to use internal hyperlink target reference for avoid the situation when the page move to somewhere else in the future

```suggestion
The generic ``SQLExecuteQueryOperator`` can be used to execute SQL queries against an Amazon Redshift cluster using a :ref:`howto/connection:redshift`.
```

Reply:
@Taragolis Thanks